src/textual/app.py

```diff
@@ -856,7 +856,7 @@ def _register_child(self, parent: DOMNode, child: DOMNode) -> bool:
             return True
         return False
 
-    def register(
+    def _register(
```

Comment:
Devs should never need to call this directly.

There are probably a number of other methods that should be private. Will be an ongoing process to identify them